pkg/cluster/template/config/dm/prometheus.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dm
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path"
+	"text/template"
+
+	"github.com/pingcap/tiup/pkg/cluster/embed"
+)
+
+// PrometheusConfig represent the data to generate Prometheus config
+// You may need to update the template file if change this struct.
+type PrometheusConfig struct {
+	ClusterName       string
+	AlertmanagerAddrs []string
+	GrafanaAddr       string
+
+	MasterAddrs []string
+	WorkerAddrs []string
+}
+
+// NewPrometheusConfig returns a PrometheusConfig
+func NewPrometheusConfig(cluster string) *PrometheusConfig {
+	return &PrometheusConfig{
+		ClusterName: cluster,
+	}
+}
+
+// AddGrafana add an kafka exporter address
```

Comment:
```suggestion
// AddGrafana adds an kafka exporter address
```
The same change should be applied for the comments of the rest of this file.